src/pynxtools/dataconverter/writer.py

```diff
@@ -329,14 +329,14 @@ def add_units_key(dataset, path):
 
                     add_units_key(self.output_nexus[path_hdf5], path)
                 else:
-                    # consider changing the name here the lvalue can also be group!
+                    # consider changing the name here the value can also be group!
```

Comment:
Are we sure that this is not an intentional mentioning of a left-hand side value sometimes abbreviated as lvalue?